src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/HealthCertifiedPersonViewModel.swift

```diff
@@ -178,13 +158,27 @@ final class HealthCertifiedPersonViewModel {
 			healthCertifiedPerson: healthCertifiedPerson
 		)
 	}
+	
+	var topMostCell: TableViewSection {
+		if certificateReissuanceIsVisible {
```

Comment:
I just checked: if the certificate reissuance is visible, the alignment of the gradient is not correct. For the admission state it was correct. No idea why 😅  Haven't checked for booster notification as top most cell